client/app/router.js

```diff
@@ -21,7 +37,7 @@ export default Router.map(function () {
     this.route('bio');
     this.route('statement');
     this.route('press');
-    this.route('image', { path: '/image/:slug' });
+    this.route('image', { path: '/art/:slug' });
     this.route('edit', function () {
```

Comment:
it is related to analytics in that having `art` in the url will be better for analytics from an SEO perspective than having `image` there so I slipped it in here.